### ISSUE TITLE
Opensearch not recognize by default in Magento 2.4.6 EE with the command bin/setup

### ISSUE THREAD

**Issue by asanka-atapattu** [markshust/docker-magento#936]
Getting error with Magento 2.4.6 EE with the command **bin/setup**

```
[Progress: 909 / 1823]
Installing search configuration...

In SearchConfig.php line 81:
                                                                               
  Could not validate a connection to Elasticsearch. No alive nodes found in your cluster
```

Opensearch is not picking correctly.

After I add 
  --search-engine=opensearch \ in to the bin/setup-install it work fine.

**Comment by markshust** [comment]
Resolved in https://github.com/markshust/docker-magento/pull/937

**Comment by Digital-Pig-LLC** [comment]
Current instance shows the same error, even though "--search-engine=opensearch" exists in the setup-install.
I'm just debugging this; will let you know what I find.

**Comment by Digital-Pig-LLC** [comment]
This can be disregarded, but I did find something interesting. I was running from a Windows 11 Pro machine with WSL2, but not using WSL. I was actually running this from a Git Bash terminal as I dislike PowerShell (about 50 chars to do the equivalent of: tail -f *sigh*). I then switched to WSL Ubuntu and ran the oneline and it worked 100% first time around. So, worth asking what terminal folks are using.

Thanks for the absolutely awesome script!
